### ISSUE TITLE
[Bug Report] v-app-bar covers content

### ISSUE THREAD

**Issue by CisarJosh** [vuetifyjs/vuetify#9628]
### Environment
**Vuetify Version:** 2.1.9
**Vue Version:** 2.6.10
**Browsers:** Chrome 78.0.3904.97
**OS:** Mac OS 10.15.1

### Steps to reproduce
Fixed prop for v-app-bar with prominent or image

### Expected Behavior
Content should render after the app-bar

### Actual Behavior
Content is covered by app bar

### Reproduction Link
<a href="https://codepen.io/cisarjosh/pen/zYYaZYx" target="_blank">https://codepen.io/cisarjosh/pen/zYYaZYx</a>

### Other comments
I've switched the code also to just the app bar and it doesn't cover the content. I found one issue similar on discord about the padding, is there a solution?

**Comment by KaelWD** [comment]
https://vuetifyjs.com/en/components/application#application-components

**Comment by eriegz** [comment]
If you arrived at this page because you're also having this same problem, double-check that you are following the structure outlined in the "Default application markup" section of the [Vuetify documentation](https://vuetifyjs.com/en/components/application/#default-application-markup). Namely:

```
<v-app-bar app></v-app-bar>

...

<v-main>
  <v-container fluid>
    <!-- PUT YOUR MAIN CONTENT HERE, INSIDE THESE 2 PARENT ELEMENTS! -->
  </v-container>
</v-main>

...

```

Or, if that fails, try using `<v-toolbar>` instead of `<v-app-bar>` like [this person](https://stackoverflow.com/questions/51902929/vuetifyjs-toolbar-overlays-content-as-soon-as-fixed-prop-gets-added/51928096).

**Comment by symplytheo** [comment]
> If you arrived at this page because you're also having this same problem, double-check that you are following the structure outlined in the "Default application markup" section of the [Vuetify documentation](https://vuetifyjs.com/en/components/application/#default-application-markup). Namely:
> 
> ```
> <v-app-bar app></v-app-bar>
> 
> ...
> 
> <v-main>
>   <v-container fluid>
>     
>   </v-container>
> </v-main>
> 
> ...
> ```
> 
> Or, if that fails, try using `<v-toolbar>` instead of `<v-app-bar>` like [this person](https://stackoverflow.com/questions/51902929/vuetifyjs-toolbar-overlays-content-as-soon-as-fixed-prop-gets-added/51928096).

This works fine thanks... I had to add a class "pa-0" to the v-container too
